scriptExecutor.go

```diff
@@ -271,3 +275,48 @@ func buildDockerRunCommand(executionConf *executionConf) (string, error) {
 	return finalScript, nil
 
 }
+
+// Writes input vars to env file
+func writeToEnvFile(envMap map[string]string, filename string) error {
+	content, err := marshal(envMap)
+	if err != nil {
+		log.Println(util.DEVTRON, "error while marshalling the content for env file ", err)
+		return err
+	}
+	file, err := os.Create(filename)
+	if err != nil {
+		log.Println(util.DEVTRON, "error while creating env file ", err)
+		return err
+	}
+	defer file.Close()
+	_, err = file.WriteString(content + "\n")
+	if err != nil {
+		log.Println(util.DEVTRON, "error while writing env values to file ", err)
+		return err
+	}
+	file.Sync()
+	return err
+}
+
+// Filters values of env variables on the basis of type and inserts to slice of strings
+func marshal(envMap map[string]string) (string, error) {
```

Comment:
marshal does not seem suitable name for the work this method is doing, can we have name it something more aligned with the work it is doing?